### PR TITLE
Improved

### PR DESCRIPTION
**Pull request by ParichoyNandi**
Added Header, Footer
Fonts ,
Table
Forms
align

### CODE REVIEW

File: client/javascript/views/addpatient.ejs [hunk 1]
```diff
@@ -16,59 +16,84 @@
...
-      <select class="form-select" name="docType">
+      <select class="form-select" name="docType" required>
         <option value="patient">Patient</option>
         <option value="donor">Donor</option>
       </select>
     </div>
     <div class="col-md-2">
       <label for="name" class="form-label">ID</label>
-      <input type="text" class="form-control" name="PID">
+      <input type="text" class="form-control" name="PID" required>
```
Comment (SATYAJIT1910) [inline client/javascript/views/addpatient.ejs]: Could you add a javascript validation that no special characters are allowed?
client-side javascript to do that -

```
        function alpha(e) {
    var k;
    document.all ? k = e.keyCode : k = e.which;
    return ((k > 64 && k < 91) || (k > 96 && k < 123) || k == 8 || k == 32 || (k >= 48 && k <= 57));
        }
```

File: client/javascript/views/addpatient.ejs [hunk 2]
```diff
@@ -16,59 +16,84 @@
...
       <label for="name" class="form-label">First Name</label>
-      <input type="text" class="form-control" name="firstName">
+      <input type="text" class="form-control" name="firstName" required>
     </div>
-     <div class="col-md-4">
-      <label for="name" class="form-label">First Name</label>
-      <input type="text" class="form-control" name="firstName">
+     <div class="col-md-3">
+      <label for="name" class="form-label">Middle Name</label>
+      <input type="text" class="form-control" name="middleName">
```
Comment (SATYAJIT1910) [inline client/javascript/views/addpatient.ejs]: There is no provision made in the backend to handle middle names.

File: client/javascript/views/addpatient.ejs [hunk 3]
```diff
@@ -16,59 +16,84 @@
...
     </div>
-    <div class="col-md-2">
-      <label for="name" class="form-label">Age</label>
-      <input type="text" class="form-control" name="age">
     </div>
+    <div class="row">
     <div class="col-md-2">
       <label for="name" class="form-label">Phone</label>
-      <input type="text" class="form-control" name="phoneNumber" max="10">
+      <input type="text" class="form-control" name="phoneNumber" max="10" required>
```
Comment (SATYAJIT1910) [inline client/javascript/views/addpatient.ejs]: Make a provision that only numbers are allowed.

File: client/javascript/views/addpatient.ejs [hunk 4]
```diff
@@ -16,59 +16,84 @@
...
+      <ul class="navbar-nav mr-auto">
+        <li class="nav-item active px-2">
+          <a class="nav-link" href="/hos/registerpatient"><h3>Add a Patient or Doner</h3></a>
+        </li>
+        <li class="nav-item px-2">
+          <a class="nav-link" href="/hos/match"><h3>Find a match</h3></a>
+        </li>
+        
+        <li class="nav-item px-2">
+          <a class="nav-link" href="#"><h3>About</h3></a>
```
Comment (SATYAJIT1910) [inline client/javascript/views/addpatient.ejs]: About page required , with backend route

File: client/javascript/views/addpatient.ejs [hunk 5]
```diff
@@ -79,26 +104,73 @@
         <option value="AB-">AB-</option>
       </select>
     </div>
+    </div>
+    <div class="row">
+    <div class="col-md-2">
+      <label for="name" class="form-label">Age</label>
+      <input type="text" class="form-control" name="age" required>
```
Comment (SATYAJIT1910) [inline client/javascript/views/addpatient.ejs]: Make a provision that age must be within range of >1 to <100

File: client/javascript/views/patientlist_hos.ejs [hunk 6]
```diff
@@ -64,11 +90,52 @@
...
+        <div class="col-md-3 col-lg-4 col-xl-3 mx-auto mb-4">
+          <h6 class="text-uppercase fs-4">
+            <i class="fas fa-gem me-3 text-secondary"></i>Indriya
+          </h6>
+          <p>
+            "Connecting people to Save life"<br> We connect the organ doners and patients. In order to save lives and making a helpful ecosystem.
+          </p>
+        </div>
 
+
```
Comment (SATYAJIT1910) [inline client/javascript/views/patientlist_hos.ejs:115]: Make the footer dynamic.
Follow: https://stackoverflow.com/a/60585266/14859542

**Review by SATYAJIT1910 — CHANGES_REQUESTED** [review]
I have reviewed your code changes. Please perform the changes as per the review comments.

File: client/javascript/views/addpatient.ejs (hunk 3 above)
Comment (ParichoyNandi) [reply client/javascript/views/addpatient.ejs:77]: done

Comment (ParichoyNandi) [reply client/javascript/views/addpatient.ejs]: done

File: client/javascript/views/addpatient.ejs (hunk 1 above)
Comment (ParichoyNandi) [reply client/javascript/views/addpatient.ejs]: done

File: client/javascript/views/addpatient.ejs (hunk 5 above)
Comment (ParichoyNandi) [reply client/javascript/views/addpatient.ejs]: doing it 5 to 70

Comment (SATYAJIT1910) [reply client/javascript/views/addpatient.ejs]: fine

**Review by SATYAJIT1910 — APPROVED** [review]

File: client/javascript/views/patientlist_hos.ejs (hunk 6 above)
Comment (SATYAJIT1910) [reply client/javascript/views/patientlist_hos.ejs:115]: Work needed. Resolved for now.

**Pull request merged**